pyproject.toml

```diff
@@ -3,23 +3,21 @@ name = "hmrc_fpo_categorisation_api"
 description = "FPO Parcel Item Categorisation API"
 version = "0.1.0"
 dependencies = [
-    "numpy",
-    "sentence-transformers",
-    "torch",
-    "tqdm",
-    "fastapi",
-    "uvicorn",
-    "tqdm",
-    "aws-lambda-powertools",
-    "jmespath",
-    "prettytable",
-    "fnv-c"
+    "sentence-transformers", # Model/embedding generation
+    "torch", # Model/embedding generation
+    "fnv-c", # Hashing of cached data keys during embedding generation
+    "tqdm", # Benchmarking
+    "prettytable", # Benchmarking
+    "toml"
 ]
 
 [project.optional-dependencies]
 dev = [
+    "aws-lambda-powertools",
```

Comment:
This ends up being a development, only, dependency since its a requirements_lambda.txt requirement and only needed to run the tests of the lambda